src/styles/generic/_shared.scss

```diff
@@ -62,3 +62,13 @@ var {
   -webkit-hyphens: none;
   hyphens: none;
 }
+
+// Ensures deep linking to headers has enough space to account for the
+// top navigation so they don't overlap.
+*:not([href])[id]::before {
```

Comment:
Gross. I love it.

Reply:
Several thoughts:

1) this should at least be `pointer-events: none` so it can't accidentally prevent clicks on something above it
2) can we use a sass variable for `65px` e.g. `$header-size`
3) could the pseudo-element just be a simple `position: absolute` element which is `top: -65px`? I suppose that then requires the heading to be `position: relative`, so, ... hm.
4) we could avoid all this by having the header disappear on scroll, but that's a big design change.

Reply:
I've added the first two suggestions. I wasn't sure about making headers disappear or making them all relative...